### ISSUE TITLE
Contrast issue

### ISSUE THREAD

**Issue by vitalybaev** [robertrossmann/vscode-remedy#16]
Hi, great theme, but when it's hard to read the following case:

<img width="608" alt="CleanShot 2020-10-16 at 16 48 50@2x" src="https://user-images.githubusercontent.com/724423/96266660-8fca9f80-0fcf-11eb-92cf-745ecfab37cf.png">

**Comment by robertrossmann** [comment]
Hi, thanks for the report, I did notice this as well that it becomes quite difficult... I'll take a look and see what I can do. 🤔

**Comment by robertrossmann** [comment]
I eventually settled on this, I think it's a good enough improvement. 🚀 🎉 Will be released soon.

![improved list focus background](https://user-images.githubusercontent.com/3058150/96472500-d5d26e00-1230-11eb-98ac-03bd12b06e78.png)

I also tried using a different, contrasting colour for the matched text but nothing worked quite well as I had hoped for. So I reduced the colour of the row highlight, instead.
